tests/commands/pootle_runner.py

```diff
@@ -20,7 +20,11 @@ def test_pootle_noargs(capfd):
     """Pootle no args should give help"""
     call(['pootle'])
     out, err = capfd.readouterr()
-    assert "Type 'pootle help <subcommand>'" in out
+    # Expcted:
+    #   Type 'pootle help <subcommand>'
+    # but 'pootle' is 'pootle-script.py' on Windows
```

Comment:
I guess we need to revisit this on a separate PR. TTK required to append the `.py` on previous versions, but on 2.0.0 it doesn't need it anymore.